src/components/molecules/TablesUserList/index.ts

```diff
@@ -1 +1,6 @@
+export { TablesUserList } from "./TablesUserList";
+
+/**
+ * @deprecated use named export instead
+ */
 export { default } from "./TablesUserList";
```

Comment:
It's only used in 3 files, maybe it worth changing it there?
![image](https://user-images.githubusercontent.com/63194656/124873553-4abc2280-dfcf-11eb-89ad-0a95cd87379b.png)

Reply:
I'll call this NB, but probably happy to do so given it will be an easy change.

Reply:
fixed in 961b90e0fe910b273a48ad52c520b92132a3b7a2